src/components/Modal.tsx

```diff
@@ -40,7 +41,7 @@ function ModalOverlay(props: UniversalProps) {
     const enteredExpirationDate = todoExpirationtRef.current!.value;
 
     if (
-      enteredText.trim().length === 0 ||
+      !enteredText.trim().length ||

```

Comment:
empty string - false, try to simplify

Reply:
don't understand how can i simplify it more, with **!enteredText.trim().length** i'm cheking for entered spaces

Reply:
do not use .length. coz !enteredText.trim().length  and !enteredText.trim() will return the same false coz  "" (empty string) is false and !enteredText.trim().length   will return 0 which is also false

Reply:
aa, ok, thanks